packages/runtime/src/widgets/Flex.tsx

```diff
@@ -21,9 +24,11 @@ export type FlexProps = {
   EnsembleWidgetProps<FlexStyles & FlexboxStyles>;
 
 export const FlexWidget: React.FC<FlexProps> = (props) => {
+  const { values } = useRegisterBindings({ ...props }, props.id);
```

Comment:
Only pass in the props that belong to the Flex widget here, otherwise it may create issues with the nested widgets.